decidim-accountability/app/controllers/decidim/accountability/admin/results_controller.rb

```diff
@@ -56,8 +56,32 @@ def destroy
           redirect_to results_path(parent_id: result.parent_id)
         end
 
+        def proposals
+          respond_to do |format|
+            format.html do
+              render partial: "proposals"
+            end
+            format.json do
+              query = Decidim.find_resource_manifest(:proposals)
+                             .try(:resource_scope, current_feature)&.order(title: :asc)
+              term = params[:q]
+              if term&.start_with?("#")
+                term.delete!("#")
+                query = query.where("CAST(id AS TEXT) LIKE ?", "#{term}%")
+              else
+                query = query.where("title ilike ?", "%#{params[:q]}%")
+              end
+              render json: query.all.collect { |p| [p.title, p.id] }
+            end
+          end
+        end
+
+        #-----------------------------------------------------------------------
```

Comment:
Remove this

Reply:
why? It facilitates locating code inside artifacts!

Reply:
@tramuntanal It's because we try to conform to the current code style conventions instead of changing random bits of code to conform to our personal taste. This style, useful or not, is not used anywhere else in the whole code base.

Reply:
I couldn't have expressed it better, thanks @deivid-rodriguez 😄

Reply:
This is now not needed as there's nothing in the middle haha

Reply:
This is now not needed since there's nothing in the middle haha

Reply:
@tramuntanal remember to remove this :)